Library/Tests/ViewModels/ShareViewModelTests.swift

```diff
@@ -62,10 +62,9 @@ internal final class ShareViewModelTests: TestCase {
 
     XCTAssertEqual(["Showed Share Sheet", "Project Show Share Sheet"], self.trackingClient.events)
 
-    self.vm.inputs.shareActivityCompletion(activityType: nil,
-                                           completed: false,
-                                           returnedItems: nil,
-                                           activityError: nil)
+    self.vm.inputs.shareActivityCompletion(
+      with: .init(activityType: nil, completed: false, returnedItems: nil, activityError: nil)
+    )
```

Comment:
ill comment on this later when i get to the view model, but i upgraded this input to be a struct instead of a bunch of arguments.